src/app/cluster/details/external-cluster/external-machine-deployment-details/template.html

```diff
@@ -100,6 +100,17 @@
           </div>
         </km-property>
 
+        <ng-container *ngIf="machineDeployment.cloud.aks">
+          <km-property>
+            <div key>Provisioning State</div>
+            <div value>{{machineDeployment.phase.aks.provisioningState}}</div>
```

Comment:
Editor warns to as phase is not returning doesn't have `aks` **optional** property add property in `ExternalClusterMDPhase` for aks provider and Add node this property is used only for AKS as Azure have this power and provisioning state thing only.